kaldb/src/main/java/com/slack/kaldb/config/KaldbConfig.java

```diff
@@ -23,6 +23,9 @@
 public class KaldbConfig {
   public static Duration DEFAULT_START_STOP_DURATION = Duration.ofSeconds(15);
 
+  public static String SEARCH_METADATA_STORE_PATH = "/search";
```

Comment:
Is there a reason we don't expose these as actual configs?

Reply:
Since these are internal paths to KalDb, making these paths configurable is not very valuable. The zk path prefix is the only user exposed config at the moment.

Reply:
Ah, I forgot these would be behind the prefix as well. 👍

Reply:
These probably should be suffixed with `...STORE_ZK_PATH` instead of just `...STORE_PATH`.

https://github.com/slackhq/kaldb/pull/117#discussion_r710507760

Reply:
Changed.